practice/hello.c

```diff
@@ -8,5 +8,5 @@
 
 #include <stdio.h>
 int main () {
-	printf ("Hello,World");

```

Comment:
This change does not belong in this assignment. You can delete this file and push again.